src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/LicenseRewriteRule.scala

```diff
@@ -93,7 +96,7 @@ case class LicenseRewriteRule(cfgDir: File) extends RewriteRule with DebugEnhanc
     } catch {
       case e: Exception =>
         logger.error("Invalid license URI: {}", licenseText, e)
-        throw new IllegalArgumentException("Not a valid license URI", e)
+        throw e
```

Comment:
Causes an InvalidBagException which is logged, no need to log here too.

https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/04c9c80d60e933d84017873d2dc6c961794654e9/src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/DdmTransformer.scala#L185-L188

https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/1f8a295db32d3972a42746c1ba45fec6c95a88ec/src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala#L113-L114